domain/machine/state/machine_cloud_instance.go

```diff
@@ -282,7 +282,7 @@ func (st *State) GetInstanceStatus(ctx context.Context, mName machine.Name) (sta
 	// frequent retrieval).
 	machineStatusParam := machineStatusWithData{}
 	statusCombinedQuery := `
-SELECT (st.status,
+SELECT (st.status_id,
 		st.message,
```

Comment:
Any chance you can fix this formatting here please?

Reply:
yup